frontend/src/ResourceTreeNode.svelte

```diff
@@ -129,9 +144,17 @@
       resourceNodeDataMap[self_id].commentsPromise =
         rootResource.get_comments();
     }
+    if (resourceNodeDataMap[self_id].modified === undefined) {
+      resourceNodeDataMap[self_id].modified = false;
+    }
+    if (resourceNodeDataMap[self_id].prevModified === undefined) {
+      resourceNodeDataMap[self_id].prevModified = false;
+    }
```

Comment:
The names `modified` and `prevModified` are not clear. What is the difference? Presumably anything modified has been previously modified?